README.md

```diff
@@ -75,11 +77,34 @@ Expected output:
 - metadata files describing each workflow step (yml format)
 - log files (stdout and stderr; ascii format)
 
+#### Workflow with database access
+
+To run workflows which require access to the model database, use the following commands:
+
+```bash
+cwltool --custom-net bridge \
+    --preserve-environment DB_API_PORT \
+    --preserve-environment DB_SERVER \
+    --preserve-environment DB_API_USER \
+    --preserve-environment DB_API_PW \
+    --preserve-environment DB_API_AUTHENTICATION_DATABASE  \
+    SetMirrorPanelRandomReflection.cwl ../tests/resources/test_derive_mirror_panel_rnda.yml
```

Comment:
I probably should know this, but it's easier to ask. You provide the file `../tests/resources/test_derive_mirror_panel_rnda.yml` at the end of the command without any option like `--input`, is that correct?

Reply:
yes, these are two positional arguments:
```
positional arguments:
  cwl_document          path or URL to a CWL Workflow, CommandLineTool, or ExpressionTool. If the `inputs_object` has a `cwl:tool` field indicating the
                        path or URL to the cwl_document, then the `cwl_document` argument is optional.
  inputs_object         path or URL to a YAML or JSON formatted description of the required input values for the given `cwl_document`.
```